mappings/com/mojang/blaze3d/platform/GlStateManager.mapping

```diff
@@ -176,6 +176,11 @@ CLASS net/minecraft/class_4493 com/mojang/blaze3d/platform/GlStateManager
 	METHOD method_21972 multMatrix (Ljava/nio/FloatBuffer;)V
 		ARG 0 matrix
 	METHOD method_21973 initFramebufferSupport (Lorg/lwjgl/opengl/GLCapabilities;)Ljava/lang/String;
+		COMMENT Configures the frame buffer and populates {@link FrameBufferInfo} with the appropriate constants
+		COMMENT for the current GLCapabilities.
+		COMMENT
+		COMMENT @returns A human-readable string representing the selected frame buffer technology.
```

Comment:
We need to be consistent here. I propose we use phrases instead of full sentences for jd tags.